MIP0/mip0.md

```diff
@@ -7,287 +11,254 @@ Title: The Maker Improvement Proposal Framework
-List of of personnel currently occupying core roles.
+- **Maker Improvement Proposals (MIPs):** - **Maker Improvement Proposals (MIPs):** MIPs are standardized documents subject to voting that (once enacted) regulate and define the behavior of the Maker Governance and the Maker Protocol. MIPs can be either general or technical (see `MIP0c4`). MIPs can be added, amended, replaced, and removed.
+-   **MIP Sets:** MIP Sets are groups of interdependent MIPs that work as a modular whole. See `MIP0c12`.
+-   **MIP Types:** MIPs can be either general, technical, or process MIPs. See `MIP0c5`.
+-   **Subproposals (SPs):** Subproposals are instances of Process MIP Components. See `MIP0c5`.  
+-   **Minimum Feedback Period:** The minimum amount of time within which the community is able to give feedback in response to a proposed MIP.  
+-   **Minimum Frozen Period:** The minimum amount of time during which a MIP must remain unchanged (frozen) before it can be submitted for ratification/implementation.
+-   **Governance Facilitator(s):** Governance Facilitators are tasked with ensuring a smooth operation of the governance process. This involves a wide range of activities, anything from general administration through signals gathering to governance scheduling. Governance Facilitators cannot delegate their responsibilities in the MIP Framework and process that require their formal involvement.
+-  **MIP Editor(s):** MIP Editors enforce the administrative and editorial aspects of the overall MIPs process and program. Governance Facilitators are granted the MIP Editor role, but can also delegate the MIP Editor role to contributors.
+-   **Domain Teams**: Domain Teams work for the DAO, are onboarded through governance and are paid by the Protocol. Domain teams perform defined duties for the DAO, such as overseeing critical processes and mitigating risk. These teams consist of but are not limited to Risk, Oracles, Smart Contracts or Legal.
 
-**MIP0c10: MIP Editor Role**  
-A component that defines the responsibilities, criteria and grounds for removal of the MIP Editor role.
 
-**MIP0c11: Governance Facilitator Role**  
-A component that defines the responsibilities, criteria and grounds for removal of the Governance Facilitator role.
+---
+### MIP0c2: Core Principles
 
-**MIP0c12: Core Personnel Onboarding**  
-A process component that defines the process to add personnel to the MIP Editor or Governance Facilitator roles.
+1. **Specificity:** A MIP needs to define and address a specific behavior or single responsibility. MIPs covering many different behaviors or responsibilities will not be allowed and must be split up into multiple proposals.
 
-**MIP0c13: Core Personnel Offboarding**  
-A process component that defines the process to remove personnel from the MIP Editor or Governance Facilitator roles.
+2. **Completeness:** A MIP or a MIP Set is complete if it succesfully covers all of the necessary parts of a given behavior.
```

Comment:
Not 100% happy with this one, but not sure how to change it.

Reply:
Maybe "A MIP should be as self-contained and self-sufficient as possible"?